src/utils/actions.js

```diff
@@ -100,6 +100,18 @@ const actions = [
     },
     requiredInput: ['field', 'value', 'softLock'],
   },
+  {
+    value: 'selectMatchingValues',
+    translation: '$Select matching values in a field',
```

Comment:
Needs proper translation string

Reply:
Yup. started the discussion in the PS channel to come up with a good name.